src/middlewared/middlewared/plugins/filesystem_/acl.py

```diff
@@ -100,25 +99,7 @@ def path_get_acltype(self, path):
         if path_location(path) is FSLocation.EXTERNAL:
             raise NotImplementedError
 
-        try:
-            os.getxattr(path, "system.posix_acl_access")
-            return ACLType.POSIX1E.name
-
-        except OSError as e:
-            if e.errno == errno.ENODATA:
-                return ACLType.POSIX1E.name
-
-            if e.errno != errno.EOPNOTSUPP:
-                raise
-
-        try:
-            os.getxattr(path, "system.nfs4_acl_xdr")
-            return ACLType.NFS4.name
-        except OSError as e:
-            if e.errno == errno.EOPNOTSUPP:
-                return ACLType.DISABLED.name
-
-            raise
+        return path_get_acltype(path)
 
     @accepts(
```

Comment:
We'll need `@api_method(FilesystemChownArgs, FilesystemChownResult)` unless this will be handled in a future pr

Reply:
`filesystem.setperm` and `filesystem.can_access_as_user` should also be converted

Reply:
I was planning on this for follow-on PR. This one was already quite large.